Shared/Extensions/StringExtensions.swift

```diff
@@ -154,4 +154,24 @@ extension String {
         
         return textEntered
     }
+    
+    // Taken from https://gist.github.com/pwightman/64c57076b89c5d7f8e8c
+    public var javaScriptEscapedString: String {
```

Comment:
is this necessary if https://github.com/brave/brave-ios/blob/90359d702714535cbc9fb743cdef8480667a78a3/Shared/Extensions/WKWebViewExtensions.swift#L25 does JS escaping when escapeArgs is set?

cc @jumde

Reply:
We tested it today with @jumde, and this is necessary, as explanation in comment or in the gist says
https://gist.github.com/pwightman/64c57076b89c5d7f8e8c